_examples/real-world-examples/server-sent-events/server/models.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"regexp"
+	"strings"
+	"time"
+)
+
+// Note that in a real application using both "json" and "bson" tags in the same structure is strongly discouraged.
+// We use common models for database storage and HTTP API just to make this example simple and easy to grasp.
```

Comment:
maybe link to DRY art?

Reply:
c4eba92ae51e6cb3656a235ee014acfb11293212